libvips/foreign/webpsave.c

```diff
@@ -84,6 +84,14 @@
 #include <webp/types.h>
 #include <webp/mux.h>
 
+/* By default, entropy passes is equal to one, but 
+ * if target_size is provided, this is a more reasonable
+ * default value.
+ */
+#ifndef TARGET_SIZE_DEFAULT_PASSES
+#define TARGET_SIZE_DEFAULT_PASSES 3
```

Comment:
I don't mind a magic number for this. And it needs to be explained to the user in the docs, so it's not really a symbolic value.